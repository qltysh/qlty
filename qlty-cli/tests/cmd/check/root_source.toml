bin.name = "qlty"
args = ["check", "--all", "--verbose", "--no-cache"]
