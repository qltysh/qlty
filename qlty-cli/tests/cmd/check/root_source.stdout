 JOBS: 1 

Plugin  Result   Targets   Time   Debug File
simple  Success  1 target  [..]s  .qlty/out/invoke-[..].yaml

Checked 1 file
